R/gcalendr_auth.R

```diff
@@ -156,57 +147,78 @@ calendar_has_token <- function() {
   inherits(.auth$cred, "Token2.0")
 }
 
-#' View or edit auth config
+#' Edit and view auth configuration
```

Comment:
Syncing you up with what I've done elsewhere. This needed to become a more focused setter and less of a getter.